nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/WriteAheadFlowFileRepository.java

```diff
@@ -202,15 +191,8 @@ public void initialize(final ResourceClaimManager claimManager, final Repository
         // delete backup. On restore, if no files exist in partition's directory, would have to check backup directory
         this.serdeFactory = serdeFactory;
 
-        // The specified implementation can be plaintext or encrypted; the only difference is the serde factory
-        if (walImplementation.equals(SEQUENTIAL_ACCESS_WAL) || walImplementation.equals(ENCRYPTED_SEQUENTIAL_ACCESS_WAL)) {
-            // TODO: May need to instantiate ESAWAL for clarity?
-            wal = new SequentialAccessWriteAheadLog<>(flowFileRepositoryPaths.get(0), serdeFactory, this);
-        } else {
-            throw new IllegalStateException("Cannot create Write-Ahead Log because the configured property '" + NiFiProperties.FLOWFILE_REPOSITORY_WAL_IMPLEMENTATION +
-                    "' has an invalid value of '" + walImplementation + "'. Please update nifi.properties to indicate a valid value for this property.");
-        }
-
+        // TODO: May need to instantiate ESAWAL for clarity?
```

Comment:
The `TODO` comment no longer applies, it was referring to the encrypted implementation.